components/bms_worker/include/Sys.h

```diff
@@ -0,0 +1,79 @@
+/**
+ * @file Sys.h
+ * @brief  Header file for System Manager
+ * @author Joshua Lafleur (josh.lafleur@outlook.com)
+ * @date 2023-12-27
+ */
+
+#pragma once
+
+/******************************************************************************
+ *                             I N C L U D E S
+ ******************************************************************************/
+
+/**< Firmware Includes */
+#include "HW_gpio.h"
+
+/******************************************************************************
+ *                              D E F I N E S
+ ******************************************************************************/
+
+/******************************************************************************
+ *                              E X T E R N S
+ ******************************************************************************/
+
+/******************************************************************************
+ *                             T Y P E D E F S
+ ******************************************************************************/
+
+typedef uint8_t BMS_Errors_t; 
+
+typedef enum {
+    BMS_ERROR = 0x00,
```

Comment:
the part of the enum that is different should be at the end, so the majority of each one is consistent. Also, namespacing. Eg. in this case you could do SYS_ERROR_BMS, SYS_ERROR_CELL_TEMP, etc.